conftest.py

```
@@ -1150,6 +1150,16 @@ def prepared_plan(
                         "Static IP verification may not work for this VM."
                     )
 
+                try:
+                    detect_guest_nic_names(
+                        source_provider=source_provider,
+                        vm=provider_vm_api,
+                        source_provider_data=fixture_store["source_provider_data"],
+                        vm_details=source_vm_details,
+                    )
+                except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)